templates/rhdm70-kieserver.yaml

```diff
@@ -15,141 +15,141 @@ message: A new Decision Manager Execution Server application has been created in
   project. Please be sure to create the "kieserver-service-account" service account and
   the secret named "${HTTPS_SECRET}" containing the ${HTTPS_KEYSTORE} file used for
   serving secure content.
+  Only stateless API calls to the Decision Server are supported.
 parameters:
 - displayName: Application Name
   description: The name for the application.
   name: APPLICATION_NAME
   value: myapp
   required: true
 - displayName: EAP Admin User
-  description: EAP administrator username
+  description: EAP administrator user name. Use this user account if you need to log on to the JBoss EAP console.
   name: ADMIN_USERNAME
   value: eapadmin
   required: false
 - displayName: EAP Admin Password
-  description: EAP administrator password
+  description: EAP administrator password.
   name: ADMIN_PASSWORD
   from: "[a-zA-Z]{6}[0-9]{1}!"
   generate: expression
   required: false
-- displayName: KIE Admin User
```

Comment:
There is a specific user to remove this?
Or this user is intend to be used only with the workbench?

Reply:
@sutaakar told me this is intended only for use with the workbench, so can be removed on templates that do not include the workbench.

Made changes for other comments and pushed them now. As for #15, I will replicate changes in the new no-https template when it is merged.

Reply:
Awesome, thanks.